Dockerfile

```diff
@@ -17,6 +14,14 @@ RUN mv ./kubectl /usr/local/bin/kubectl && chmod +x /usr/local/bin/kubectl
 
 # Add scripts for Istio
 COPY scripts /usr/local/app/scripts/
-RUN chmod +x /usr/local/app/scripts/init_kubeconfig.sh /usr/local/app/scripts/run.sh /usr/local/app/scripts/create_istio_system.sh /usr/local/app/scripts/uninstall_istio_system.sh /usr/local/app/scripts/get_grafana_dashboards.sh
+RUN chmod +x /usr/local/app/scripts/init_kubeconfig.sh /usr/local/app/scripts/run.sh /usr/local/app/scripts/create_istio_system.sh /usr/local/app/scripts/uninstall_istio_system.sh /usr/local/app/scripts/get_grafana_dashboards.sh /usr/local/app/scripts/setup_release_mirror.sh /usr/local/app/scripts/generate_ssl.sh
 RUN mkdir -p /usr/local/app/dashboards && /usr/local/app/scripts/get_grafana_dashboards.sh
+
+# Add nginx configuration
+COPY nginx.conf /etc/nginx/nginx.conf
+
+# Get Istio tar for nginx
+RUN mkdir -p /usr/share/nginx/html/istio/istio/releases/download/${ISTIO_VERSION}
+RUN curl -L https://github.com/istio/istio/releases/download/${ISTIO_VERSION}/istio-${ISTIO_VERSION}-osx.tar.gz -o /usr/share/nginx/html/istio/istio/releases/download/${ISTIO_VERSION}/istio-${ISTIO_VERSION}-osx.tar.gz
```

Comment:
I needed to use the `linux-amd64` artefact in my tests, with the osx ones the upgrade failed.

Reply:
This will be resolved by switching to your pull image scripts